streaming/src/main/scala/org/apache/spark/streaming/util/StateMap.scala

```diff
@@ -77,6 +80,7 @@ private[streaming] class EmptyStateMap[K: ClassTag, S: ClassTag] extends StateMa
 }
 
 /** Implementation of StateMap based on Spark's [[org.apache.spark.util.collection.OpenHashMap]] */
+@DefaultSerializer(classOf[KryoJavaSerializer])
```

Comment:
Can you explain what this does? Just for some context?
Also will this serialize using Kryo or just embed Java serialization within Kryo? Maybe it is a good idea to make it as efficient in Kryo as possible.